projects/control-service/projects/base/build.gradle

```diff
@@ -78,6 +78,7 @@ dependencies {
    testImplementation versions.'org.awaitility:awaitility'
    testImplementation versions.'com.github.tomakehurst:wiremock'
    testImplementation versions.'org.mockito:mockito-core'
+   testImplementation versions.'net.bytebuddy:byte-buddy'
```

Comment:
don't see an import change; where do you use the newly introduced library?

Reply:
In fact this fixes an issue that was introduced into master yesterday where the latest version of mockito causes tests to fail because they need this jar.